### ISSUE TITLE
[Feb 2025] Accessibility Audit: Static site Getting started section

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8520]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the pages in the "Getting started" section of the static site that have been updated in the last 3 months. Do the page scan as well as any relevant guided tests.
Pages to visit:

- [x] https://dev3.simplereport.gov/getting-started/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/onboard-your-organization/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/where-does-simplereport-work/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/test-devices-we-support/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/activate-your-user-account/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/log-in-to-simplereport/
- [x] https://dev3.simplereport.gov/getting-started/organizations-and-testing-facilities/get-training/
- [x]  https://dev3.simplereport.gov/getting-started/public-health-departments/about-prime-reportstream/
- [x] https://dev3.simplereport.gov/getting-started/public-health-departments/simplereport-data-catalog/

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5766, #6326

**Comment by emyl3** [comment]
Findings (false positives have a new column `Note` or `Notes` 😅 with an explanation on why it was a false positive:
- `/getting-started` - [202502-dev.simplereport.gov_getting-started.json](https://github.com/user-attachments/files/18970231/202502-dev.simplereport.gov_getting-started.json)
